addons/mail/static/src/models/message/message.js

```diff
@@ -368,6 +368,18 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        /**
+         * @returns {string}
+         */
+        _computeCleanedSubject() {
+            if (this.subject) {
+                const prefix = ['RE:', 'Re:', 'FW:', 'FWD:'];
+                let cleanedSubject = this.subject;
+                prefix.forEach((pref) => cleanedSubject = cleanedSubject.replace(pref, ''));
+                return cleanedSubject.trim();
+            }
```

Comment:
As I already requested, this should only remove the prefix at the _start_ of the subject. Using `replace` will remove the first occurrence, even if it's in the middle, which is not what we want.

Pseudo code:
```
while cleanedSubject starts (index of 0) with any prefix (case insensitive -> cleaned subject can be made all lowercase and prefix too):
    remove prefix from start of cleanedSubject, also remove extra white-space at the the start
```